downstream/modules/platform/con-aap-upgrade-from-21.adoc

```diff
@@ -1,11 +1,15 @@
 
 
-[id="aap-upgrade-from-21_{context}"]
+[id="aap-upgrades_{context}"]
 
-= Upgrading from {PlatformNameShort} 2.1
+= {PlatformNameShort} upgrades
 
-Upgrade from {PlatformNameShort} version 2.1 to version {PlatformVers} by following all the procedures in these sections:
+Upgrading to version {PlatformVers} from {PlatformNameShort} 2.1 or later involves downloading the installation package and then performing the following steps:
 
+* Set up your inventory to match your installation environment.
+* Run the {PlatformVers} installation program over your current {PlatformNameShort} installation.
+* Import or generate a new {HubName} API token.
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I'm a big, big fan of the additional resources section. Using inline links adds noise to the content and risks getting users off track.

The additional resources section means that readers are much more likely to go through the full text in the topic - and not miss anything essential. It also trains the reader to look for links in the same place. After a bit of navigation, readers generally know where to go to find their link.